test/x3/actions.cpp

```diff
@@ -65,9 +67,9 @@ int main()
     }
 
     {
-        using namespace std::placeholders;
+        using namespace boost::placeholders;
         char const *s1 = "{42}", *e1 = s1 + std::strlen(s1);
-        x3::parse(s1, e1, '{' >> int_[std::bind(fun_action(), _1)] >> '}');
+        x3::parse(s1, e1, '{' >> int_[boost::bind(fun_action(), _1)] >> '}');
```

Comment:
With std::bind it does not compile, it static_assert-s "tuple index out of bounds". I've tried to remove this static_assert - compiles, works. Can rollback.